concent_api/core/tests/test_core_views.py

```diff
@@ -102,88 +108,99 @@ def test_send_should_return_http_200_if_message_timeout(self):
         assert Message.objects.count()       == 0
         assert ReceiveStatus.objects.count() == 0
 
-        message_task_to_compute            = self.message_task_to_compute
-        message_task_to_compute.deadline   = self.message_timestamp - 1
-        message_force_report_computed_task = MessageForceReportComputedTask(
+        task_to_compute = self.task_to_compute
+        task_to_compute.compute_task_def['deadline'] = self.message_timestamp - 1
+        correct_golem_data = ForceReportComputedTask(
             timestamp = self.message_timestamp,
-            message_task_to_compute = dump(
-                message_task_to_compute,
-                PROVIDER_PRIVATE_KEY,
-                REQUESTOR_PUBLIC_KEY,
-            )
         )
+        correct_golem_data.task_to_compute = task_to_compute
 
         response = self.client.post(
             reverse('core:send'),
-            data                           = dump(message_force_report_computed_task, PROVIDER_PRIVATE_KEY, settings.CONCENT_PUBLIC_KEY),
-            content_type                   = 'application/octet-stream',
-            HTTP_CONCENT_CLIENT_PUBLIC_KEY = b64encode(PROVIDER_PUBLIC_KEY).decode('ascii')
+            data = dump(
+                correct_golem_data,
+                REQUESTOR_PRIVATE_KEY,
+                CONCENT_PUBLIC_KEY),
+            content_type = 'application/octet-stream',
+            HTTP_CONCENT_CLIENT_PUBLIC_KEY = b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii')
         )
 
         self.assertEqual(response.status_code, 200)
         response_message = load(
             response.content,
-            PROVIDER_PRIVATE_KEY,
-            settings.CONCENT_PUBLIC_KEY,
+            REQUESTOR_PRIVATE_KEY,
+            CONCENT_PUBLIC_KEY,
         )
         self.assertIsInstance(response_message, GolemMessage)
 
     @freeze_time("2017-11-17 10:00:00")
     def test_send_should_return_http_400_if_data_is_incorrect(self):
-        message_force_report_computed_task_1 = MessageForceReportComputedTask(
-            timestamp = 1510911047,
-            message_task_to_compute = dump(
-                MessageTaskToCompute(),
-                PROVIDER_PRIVATE_KEY,
-                REQUESTOR_PUBLIC_KEY
-            )
+        force_report_computed_task = ForceReportComputedTask(
+            timestamp = 1510912600
         )
```

Comment:
You should add an `assert` to ensure that the timestamp is still before deadline.